DependencyInjection/PixelartSwiftmailerManipulatorExtension.php

```diff
@@ -82,4 +83,38 @@ private function configureMailerManipulator($name, array $mailer, ContainerBuild
             ;
         }
     }
+
+    /**
+     * @param string           $name      The mailer name
+     * @param array            $mailer    The mailer configuration
+     * @param ContainerBuilder $container The container builder
+     */
+    private function configureMailerImpersonation($name, array $mailer, ContainerBuilder $container)
```

Comment:
I would call it `configureMailerFromAddress`, because that's the config name we are using and configuring on the mailer. :grin: